test_setup.alerts_logger.yml

```diff
@@ -0,0 +1,41 @@
+---
+# ========================================
+#  QE configuration for alerts logger
+# ========================================
+#
+# This playbook serves as setup of alerts logger on tendrl server machine.
+# It periodically logs alerts from tendrl api based on tendrl user.
+
+- hosts: usm_server
+  remote_user: root
+  vars:
+    tendrl_user: "admin"
+    tendrl_password: "adminuser"
+
+  tasks:
+    - name: Create alerts_logger python script
+      copy:
+        dest: /usr/local/bin/alerts_logger.py
+        src: alerts_logger.py
+        mode: "+x"
+
+    - name: Create service file
+      copy:
+        dest: /etc/systemd/system/
+        src: alerts_logger@.service
+
+    - name: force systemd to reread configs
+      systemd: daemon_reload=yes
+
+    - name: Add user to users.ini
+      ini_file:
+        path: /usr/local/bin/users.ini
```

Comment:
I would rather propose some other place for the configuration file (somewhere in `/etc/...`). And also some better (more explanatory) name - something like `usmqe_alerts_logger_users.ini` or similar.